views/dashboard.handlebars

```diff
@@ -14,12 +14,11 @@
     {{#if jokes.length}}
      {{#each jokes as |joke|}}
      <div class="post">
-        <h5>{{joke.username}}'s Joke:</h5>
```

Comment:
Yes I see this, this is bringing it back to main, not sure why it changed to begin with